pkgs/development/libraries/cpputest/default.nix

```diff
@@ -0,0 +1,17 @@
+{stdenv, fetchurl}:
+
+stdenv.mkDerivation rec {
+  version = "3.7.2";
+  name = "cpputest-${version}";
+
+  src = fetchurl {
+    url = "https://github.com/cpputest/cpputest.github.io/blob/master/releases/${name}.tar.gz?raw=true";
```

Comment:
Please use this url: `https://github.com/cpputest/cpputest/releases/download/${version}/${name}.tar.gz`